website/docs/r/batch_job_definition.html.markdown

```diff
@@ -70,6 +71,12 @@ The following arguments are supported:
 
 * `attempts` - (Required) The number of times to move a job to the `RUNNABLE` status. You may specify between `1` and `10` attempts.
 
+## timeout
+
+`timeout` supports the following:
+
+* `attempt_duration_seconds` - (Required) The time duration in seconds after which AWS Batch terminates your jobs if they have not finished. The minimum value for the timeout is `60` seconds.
```

Comment:
This still says `Required` (and likewise for `attempts` above). Both are now `Optional`